src/gwt/src/org/rstudio/studio/client/workbench/views/help/HelpPane.java

```diff
@@ -161,6 +153,13 @@ else if (scrollPos_ != null)
          }
       };
       
+      prefs_.helpFontSizePoints().bind(new CommandWithArg<Double>()
```

Comment:
Is this part of this PR? Seems unrelated but I may just not be getting it.

Reply:
Yes, it caused issues when loading a view without the HelpPane since it was trying to default a window that didn't exist.